install/jaspic/docs/assertions/JASPICJavadocAssertions.html

```diff
@@ -24,7 +24,8 @@
 <body bgcolor="white">
 <br>
 <CENTER>
-<h2>Java Authentication SPI for Containers<br>JASPIC - 1.1<br>
+  <h2><!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}-->
+  <span data-sheets-value="{'1':2,'2':'Jakarta Authentication'}" data-sheets-userformat="{'2':513,'3':{'1':0},'12':0}">Jakarta Authentication</span> - 1.1<br>
```

Comment:
I don't know what all this html is for but it looks like a copy/paste problem.

Reply:
I guess I didn't use paste text only.